alien-module/control.lua

```diff
@@ -14,7 +14,7 @@ end)
 function init_gui()
 	for _, player in pairs(game.players) do
         player.gui.top.add{type="label", name="killcount", caption="Killcount"}
-		player.gui.top.killcount.caption = "Killcount: " .. global.killcount
+		player.gui.top.killcount.caption = "消灭异虫数量: " .. global.killcount
```

Comment:
Thanks for the translation but i think the english text would be preferable here